exercises/nth-prime/NthPrimeTest.fs

```diff
@@ -1,4 +1,4 @@
-// This file was auto-generated based on version 1.0.0 of the canonical data.
+// This file was auto-generated based on version 2.0.0 of the canonical data.
```

Comment:
@ErikSchierboom do you know changed here? It's weird to see the change of a major version but no change to the rest of the file.

Reply:
Well, the generator was changed: https://github.com/exercism/fsharp/pull/423/files#diff-378e45c2fa29f82b16b9d312d5f5c18aR240 The type of field used to represent an error changed.

Reply:
Ok thanks for the clarification!